app/views/docs/functions-deploy.phtml

```diff
@@ -0,0 +1,216 @@
+<p>
+    Appwrite Functions are mini-applications in Appwrite with their own endpoints. 
+    Each function can have many deployments, which can be thought of as versions of the mini-application.
+</p>
+
+<p>
+    Functions can be created and deployed in different ways to meet your unique development habits.
+    You can automatically deploy Appwrite Functions from source control, build your own deployment pipelines using the Appwrite CLI, or upload code files manually.
+    Here's everything you need to know to deploy your first Appwrite Function.
+</p>
+
+<h2><a href="#vcs" id="vcs">VCS (Version Control System)</a></h2>
+<p>
+    The recommended way to manage your Appwrite Function deployments is to use a version control system, like Git.
+    This offers simple versioning and collaboration that will easily fit into the rest of your development workflow.
+</p>
+
+<h3>Create Function</h3>
+<p>Before deploying your function with VCS, create a new function attached to your VCS repo.</p>
+<ol class="margin-top margin-bottom-large text-size-normal">
+    <li>
+        Navigate to <b>Functions</b> from the side bar of the Appwrite Console.
+    </li>
+    <li>
+        Click <b>Create function</b>.
+    </li>
+    <li>
+        When asked to <b>Choose your source</b>, under <b>Connect Git repository</b>, select your provider.
+    </li>
+    <li>
+        Search for the Git repository that hold your function and click <b>connect</b>.
+    </li>
+    <li>
+        Select a production branch. New commits made to the production branch will be <b>automatically deployed and activated</b>.
+    </li>
+    <li>
+        Input the root directory of the function inside the repository. 
```

Comment:
```suggestion
        Input the root directory of the function inside the repository. If you have only one function in your repository, you can leave this empty. If you have multiple, root directory should point to the folder of your function. This should be directory in which your custom build commands can run successfully. It also allows Appwrite build step to be more efficient and only clone what's necessary.
```

Reply:
Feel free to simplify, but I feel like we are not explaining root directory here. In short, it should point to path that has everything your function needs, but nothing extra.